### PR TITLE
Update url paths to include `-drivers` suffix for consistency

### PR DESCRIPTION
**Pull request by rachel-mack**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-ecosystem/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-30932>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-30932/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- N/A ~Did you run a spell-check?~
- N/A ~Did you run a grammar-check?~
- [x] Are all the links working?

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -212,10 +212,10 @@ raw: docs/drivers/drivers/go/ -> ${base}/go/current/
 raw: docs/drivers/drivers/java/ -> ${base}/java-drivers/
 raw: docs/drivers/drivers/kotlin/ -> ${base}/kotlin-drivers/
 raw: docs/drivers/drivers/node/ -> $${base}/node/current/
-raw: docs/drivers/drivers/php/ -> ${base}/php/
-raw: docs/drivers/drivers/python/ -> ${base}/python/
+raw: docs/drivers/drivers/php/ -> ${base}/php-drivers/
+raw: docs/drivers/drivers/python/ -> ${base}/python-drivers/
 raw: docs/drivers/drivers/pymongo/ -> ${base}/pymongo/
-raw: docs/drivers/drivers/ruby/ -> https://www.mongodb.com/docs/ruby-driver/current/
```
Comment (rachel-mack) [inline config/redirects:218]: The first item in the Ruby dropdown menu links to this site.

File: config/redirects [hunk 2]
```diff
@@ -212,10 +212,10 @@ raw: docs/drivers/drivers/go/ -> ${base}/go/current/
 raw: docs/drivers/drivers/java/ -> ${base}/java-drivers/
 raw: docs/drivers/drivers/kotlin/ -> ${base}/kotlin-drivers/
 raw: docs/drivers/drivers/node/ -> $${base}/node/current/
-raw: docs/drivers/drivers/php/ -> ${base}/php/
-raw: docs/drivers/drivers/python/ -> ${base}/python/
+raw: docs/drivers/drivers/php/ -> ${base}/php-drivers/
+raw: docs/drivers/drivers/python/ -> ${base}/python-drivers/
```
Comment (mongoKart) [inline config/redirects:216]: @ccho-mongodb These redirects don't seem to be working. In staging, I still see the old page, which now no longer shows the correct navbar entries. Do these redirects only take effect after deployment?

Comment (ccho-mongodb) [reply config/redirects:216]: Redirects aren't written to the staging environment, only the production environment. Usually to build confidence that they will work in production, we generate the redirects output with mut-redirects, and paste that into the PR. Then, we test them in production once deployed.

**Comment by rachel-mack** [comment]
Here's a diff of mut-redirects output (full txt files attached for reference):
```
140,141c140,141
< Redirect 301 /docs/drivers/drivers/php/ https://www.mongodb.com/docs/drivers/php/
< Redirect 301 /docs/drivers/drivers/python/ https://www.mongodb.com/docs/drivers/python/
---
> Redirect 301 /docs/drivers/drivers/php/ https://www.mongodb.com/docs/drivers/php-drivers/
> Redirect 301 /docs/drivers/drivers/python/ https://www.mongodb.com/docs/drivers/python-drivers/
143c143
< Redirect 301 /docs/drivers/drivers/ruby/ https://www.mongodb.com/docs/ruby-driver/current/
---
> Redirect 301 /docs/drivers/drivers/ruby/ https://www.mongodb.com/docs/drivers/ruby-drivers/
```
[redirects-new.txt](https://github.com/mongodb/docs-ecosystem/files/13243138/redirects-new.txt)
[redirects-orig.txt](https://github.com/mongodb/docs-ecosystem/files/13243140/redirects-orig.txt)

File: config/redirects [hunk 3]
```diff
@@ -226,3 +226,9 @@ raw: docs/drivers/drivers/driver-compatibility-reference -> ${base}/
 # Redirect for outdated Community Drivers page
 
 raw: docs/drivers/community-supported-drivers/ -> ${base}/
+
+# Redirects for consistent language-spcific driver landing pages
+
+raw  docs/drivers/php/ -> ${base}/php-drivers/
+raw  docs/drivers/python/ -> ${base}/python-drivers/
+raw  docs/drivers/ruby/ -> ${base}/ruby-drivers/
```
Comment (ccho-mongodb) [inline config/redirects:234]: Issue: 
I could be wrong, but I think these need a colon after "raw". These redirects also didn't surface in the mut-redirects output diff, so I am guessing they were ignored.

Suggestion:

```suggestion
raw: docs/drivers/php/ -> ${base}/php-drivers/
raw: docs/drivers/python/ -> ${base}/python-drivers/
raw: docs/drivers/ruby/ -> ${base}/ruby-drivers/
```

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
Looks mostly good -- I think some redirects are missing because of missing colons. Could you update that and the redirect output diff? Once completed, please ask for a re-review. Thanks!

**Comment by rachel-mack** [comment]
new diff:
140,141c140,141
< Redirect 301 /docs/drivers/drivers/php/ https://www.mongodb.com/docs/drivers/php/
< Redirect 301 /docs/drivers/drivers/python/ https://www.mongodb.com/docs/drivers/python/
---
> Redirect 301 /docs/drivers/drivers/php/ https://www.mongodb.com/docs/drivers/php-drivers/
> Redirect 301 /docs/drivers/drivers/python/ https://www.mongodb.com/docs/drivers/python-drivers/
143c143
< Redirect 301 /docs/drivers/drivers/ruby/ https://www.mongodb.com/docs/ruby-driver/current/
---
> Redirect 301 /docs/drivers/drivers/ruby/ https://www.mongodb.com/docs/drivers/ruby-drivers/
148a149,151
> Redirect 301 /docs/drivers/php/ https://www.mongodb.com/docs/drivers/php-drivers/
> Redirect 301 /docs/drivers/python/ https://www.mongodb.com/docs/drivers/python-drivers/
> Redirect 301 /docs/drivers/ruby/ https://www.mongodb.com/docs/drivers/ruby-drivers/

[redirects-new.txt](https://github.com/mongodb/docs-ecosystem/files/13250583/redirects-new.txt)

**Review by ccho-mongodb — APPROVED** [review]
LGTM!
Let me know if you want to pair on checking that the published version works as intended.

**Pull request merged**